pkg/reconciler/build/build.go

```diff
@@ -218,6 +218,8 @@ func (c *Reconciler) reconcile(ctx context.Context, build *buildapi.Build) error
 		build.Status.LatestAttestationImage = attestDigest
 		build.Status.Stack.RunImage = buildMetadata.StackRunImage
 		build.Status.Stack.ID = buildMetadata.StackID
+		build.Status.LifecycleVersion = buildMetadata.LifecycleVersion
+		build.Status.LifecycleCommit = buildMetadata.LifecycleCommit
```

Comment:
do we need the commit here? It doesn't seemed to be used for anything and isn't a property on the ClusterLifecycle itself

Reply:
I guess not - I'll remove it

Reply:
Updated in aea38633